test-reqs.txt

```diff
@@ -1,3 +1,5 @@
+nose
```

Comment:
No, that's correct. Just that this change doesn't relate to the pull request's title. But it's a good change nevertheless.